spt/OrangeBox/modules/ClientDLL.hpp

```diff
@@ -20,6 +21,9 @@ typedef void(__fastcall *_CViewRender__OnRenderStart) (void* thisptr, int edx);
 typedef void*(__cdecl *_GetLocalPlayer) ();
 typedef void*(__fastcall *_GetGroundEntity) (void* thisptr, int edx);
 typedef void(__fastcall *_CalcAbsoluteVelocity) (void* thisptr, int edx);
+typedef void(__fastcall * _CViewRender__RenderView) (void * thisptr, int edx, void * cameraView, int nClearFlags, int whatToDraw);
+typedef void(__fastcall * _CViewRender__Render) (void * thisptr, int edx, void * rect);
```

Comment:
Remove the whitespace between the `*` and the names like in the functions above. Also fix the whitespace in the pointers in arguments.